Dockerfile

```diff
@@ -9,6 +9,8 @@ FROM ${BASE_RUBY_IMAGE} AS builder
 
 ARG BUILD_DEPS="git gcc libc-dev make nodejs yarn postgresql-dev build-base libxml2-dev libxslt-dev ttf-ubuntu-font-family"
 
+RUN mkdir log && touch log/mail.log
+
```

Comment:
We probably don't need it in our proper production file :)

Reply:
Not 100% either way